### PR TITLE
Add management commands docs.

### LINKED ISSUE #62: Add management commands docs
They're undocumented but potentially useful for deploy processes etc.

### PR DESCRIPTION
**Pull request by nkovshov**
Fixes #62

### CODE REVIEW

File: gargoyle/management/commands/add_switch.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 class Command(BaseCommand):
     help = 'Adds or updates the specified gargoyle switch.'
+    missing_args_message = 'No switch name specified. Please provide one.'
```
Comment (adamchainz) [inline gargoyle/management/commands/add_switch.py]: please add tests for these `missing_args_message`s 👍

File: tests/testapp/test_commands.py [hunk 2]
```diff
@@ -15,9 +16,11 @@ def setUp(self):
         self.gargoyle = SwitchManager(Switch, key='key', value='value', instances=True, auto_create=True)
 
     def test_fails_with_no_arg(self):
-        with pytest.raises(CommandError):
+        with pytest.raises(CommandError) as excinfo:
             call_command('add_switch')
 
+        assert "No switch name specified. Please provide one." in six.text_type(excinfo.value)
```
Comment (nkovshov) [inline tests/testapp/test_commands.py]: Don't know if it's the best way to check an exception message

Comment (adamchainz) [inline tests/testapp/test_commands.py]: That is the best way 👍

**Comment by adamchainz** [comment]
Merged in 5e942b7b277d724b4a29b63306cd3602712a9211, thanks!

**Pull request closed without merging**